packages/main/cypress/specs/F6.cy.ts

```diff
@@ -0,0 +1,659 @@
+import { html } from "lit";
+import "@ui5/webcomponents-base/dist/features/F6Navigation.js";
+import "../../src/Button.js";
+
+describe("F6 navigation", () => {
+	describe("F6 Forward navigation", () => {
+		it("tests navigation", () => {
+			cy.mount(html`<div>
+				<div class="section">
+					<button id="before">Before element</button>
+				</div>
+				<div class="section" data-sap-ui-fastnavgroup="true">
+					<ui5-button id="first">First focusable</ui5-button>
+				</div>
+				<div class="section">
+					<ui5-button>Something focusable</ui5-button>
+				</div>
+				<div class="section" data-sap-ui-fastnavgroup="true">
+					<ui5-button id="second">Second focusable</ui5-button>
+				</div>
+				<div class="section">
+					<ui5-button>Something focusable</ui5-button>
+				</div>
+				<div class="section" data-sap-ui-fastnavgroup="true">
+					<ui5-button id="third">Third focusable</ui5-button>
+				</div>
+				<div class="section">
+					<ui5-button>After Element</ui5-button>
+				</div>
+			</div`);
+
+			// act
+			cy.get("#before").focus();
+			cy.get("#before").realPress("F6");
+
+			// assert 1st group is focused
+			cy.get("#first")
+				.should("be.focused");
+
+			// act
+			cy.get("#first").realPress("F6");
```

Comment:
once the focus is on the page, no need to chain realPress with other query command (like get) before it.